app/main.py

```diff
@@ -2,0 +1,22 @@
+from typing import List
+
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+from app.players.player import Player
+
+
+def calculate_team_total_rating(players: List[Player]) -> int:
+    sum_rating = 0
+    for player in players:
+        sum_rating += player.get_rating()
+    return sum_rating
+
+
+def elves_concert(elves: list[Elf]) -> None:
+    for elf in elves:
+        elf.play_elf_song()
+
+
+def feast_of_the_dwarves(dwarves: list[Dwarf]) -> None:
```

Comment:
The type hint `list[Dwarf]` should be replaced with `List[Dwarf]` to be consistent with the imported `List` from the `typing` module.